client/src/components/App/Content/InvestigationForm/TabManagement/ContactsAndExposurePlacesTab/NewContactDialog/NewContactDialog.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import { Close as CloseIcon, AddCircle as AddCircleIcon} from '@material-ui/icons';
+import {
+    Dialog, DialogTitle, DialogActions, Button, DialogContent,
+    FormControl, Typography, IconButton, Select, MenuItem, Divider, TextField
+} from "@material-ui/core";
+
+import useStyles from './NewContactDialogStyles';
+import Location from 'models/LocationType';
+
+const DIALOG_TITLE = 'יצירת מקום/מגע חדש';
+const LOCATION_TYPE = 'סוג אתר:';
+const CONTACT_NAME = 'שם';
+const CONTACT_ID = 'ת.ז';
+const CONTACT_PHONE = 'מספר טלפון';
+const CANCEL_LABEL = 'ביטול';
+const ADD_CONTACT = 'הוסף מגע';
+
+const NewContactDialog: React.FC<Props> = (newContactDialogProps: Props) => {
+    const [ isCreateContactEnabled, setIsCreateContactEnabled ] = useState(false);
+    const [ contactedPersonName, setContactedPersonName ] = useState('');
+    const [ contactedPersonID, setContactedPersonID ] = useState('');
+    const [ contactedPersonPhone, setContactedPersonPhone ] = useState('');
+    const [ pickedLocation, setPickedLocation ] = useState('');
+
+    const classes = useStyles();
+
+    const handleLocationPick = (event: React.ChangeEvent<{ value: unknown }>) => {
+        setPickedLocation(event.target.value as string);
+    }
+
+    const handleNameChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+        setContactedPersonName(event.target.value as string);
+    }
+
+    const handlePhoneChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+        setContactedPersonPhone(event.target.value as string);
+    }
+
+    const handleIDChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+        setContactedPersonID(event.target.value as string);
+    }
+
+    return (
+        <Dialog open={newContactDialogProps.isDialogOpen} maxWidth={false}>
+            <div className={classes.contactDialog}>
```

Comment:
Maybe you can DialogTitle here? Don't have to change it, only if it makes css easier

Reply:
DialogTitle wraps the elements inside it with margins and spaces, i preferred using a div in order that ot would seem more like the mockup